inc/saitypes.h

```diff
@@ -217,7 +218,14 @@ typedef enum _sai_object_type_t {
     SAI_OBJECT_TYPE_BRIDGE                   = 57,
     SAI_OBJECT_TYPE_BRIDGE_PORT              = 58,
     SAI_OBJECT_TYPE_TUNNEL_MAP_ENTRY         = 59,
-    SAI_OBJECT_TYPE_MAX                      = 60,
+    SAI_OBJECT_TYPE_TAM                      = 60,
+    SAI_OBJECT_TYPE_TAM_STAT                 = 61,
+    SAI_OBJECT_TYPE_TAM_SNAPSHOT             = 62,
+    SAI_OBJECT_TYPE_TAM_TRANSPORTER          = 63,
+    SAI_OBJECT_TYPE_TAM_THRESHOLD            = 64,
+    SAI_OBJECT_TYPE_TAM_HISTOGRAM            = 65,
+    SAI_OBJECT_TYPE_TAM_MICROBURST           = 66,
+    SAI_OBJECT_TYPE_MAX                      = 67,
```

Comment:
Looks as I need some help here how to proceed. Can we have a short talk today?
Please drop me how to vitalyv@marvell.com 
Thanks,
Vitaly Vovnoboy